src/css/custom.css

```diff
@@ -8,30 +8,37 @@
 
 /* Generated with https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima */
 :root {
-  --ifm-color-black: #1b1b1d;
-  --ifm-color-primary: #c63636;
-  --ifm-color-primary-dark: #be3434;
+  /* --ifm-color-black: #1b1b1d; */
+  --ifm-color-primary: #F97316;
```

Comment:
```suggestion
  --ifm-color-primary: #f97316;
  --ifm-color-primary-dark: #ee6506;
  --ifm-color-primary-darker: #e15f06;
  --ifm-color-primary-darkest: #b94f05;
  --ifm-color-primary-light: #fa8330;
  --ifm-color-primary-lighter: #fa8b3e;
  --ifm-color-primary-lightest: #fba365;
```

Reply:
Generated using https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima

We don't use all the colors for now but cleaner to have this over comments.